File: Plugins/Kingbotk/Version 2/WikiFunctions2/WikiFunctions2.vb
Public Module WikiFunctions2
    ''' <summary>
    ''' WikiFunctions2.dll version
    ''' </summary>
    Public ReadOnly Property Version() As System.Version
        Get
            Static rtn As System.Version = System.Reflection.Assembly.GetExecutingAssembly.GetName.Version
            Return rtn
        End Get
    End Property
End Module

File: AWB/Plugins/Kingbotk/Branches/Version 1/WF2CS/WikiFunctions2.cs
public static class WikiFunctions2
{
    /// <summary>
    /// WikiFunctions2.dll version
    /// </summary>
    //INSTANT C# NOTE: These were formerly VB static local variables:
    private static System.Version Version_rtn = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version;

    public static System.Version Version
    {
        get
        {
            //INSTANT C# NOTE: VB local static variable moved to class level
            //			Static rtn As System.Version = System.Reflection.Assembly.GetExecutingAssembly.GetName.Version
            return Version_rtn;
        }
    }
}